main.go

```diff
@@ -35,11 +35,15 @@ import (
 	_ "github.com/equinor/radix-api/docs"
 )
 
-const clusternameEnvironmentVariable = "RADIX_CLUSTERNAME"
+const (
+	clusternameEnvironmentVariable = "RADIX_CLUSTERNAME"
+	// requireAppConfigrationItemEnvironmentVariable = "REQUIRE_APP_CONFIGURATION_ITEM"
```

Comment:
This comment does not much explain. Also has a typo "Config*u*ration"

Reply:
Removed.